pkg/channel/distributed/common/config/observability_test.go

```diff
@@ -41,7 +43,7 @@ func TestInitializeObservability(t *testing.T) {
 
 	// Test Data
 	ctx := context.TODO()
-	metricsPort := 9876
+	metricsPort := rand.Intn(10000) + 9000
```

Comment:
was there a test issue?

Reply:
Yes, occasionally this test and the TestMetricsServer_Report would run at exactly the same time and one or the other would fail.  It was somewhat rare but I did run into it more than once.

Reply:
They could also be two unique constant ports, but since I suspect this issue arose by one test being copied from another, I was hoping to stave that off in the future.  I am open to that change instead, however.